CONTRIBUTING.md

```diff
@@ -112,9 +112,11 @@ and additionally the following rules:
   according schedule for Tumbleweed tests, e.g. in schedule/ or main.pm.
   Exceptions are any special SLE-specific behaviour or packages not in
   Tumbleweed or the case not being relevant otherwise.
-* Avoid "dead code": Don't add disabled code as nobody but you will understand
-  why this is not active. Better leave it out and keep in your local git
-  repository, either in `git stash` or a temporary "WIP"-commit.
+* Avoid "dead code": Is generally discouraged to add disabled code as others will lack
```

Comment:
Content is perfectly fine. I just suggest to consistently wrap at 80 characters per line. Consider that an implicit self-approve after wrapping the text